tests/extras_appengine_auth_models_test.py

```diff
@@ -141,14 +141,14 @@ def test_user_token(self):
         auth_id = 'foo'
 
         token = m.create_auth_token(auth_id)
-        self.assertTrue(m.validate_auth_token(auth_id, token))
+        self.assertTrue(m.validate_auth_token(auth_id, token) != None)
```

Comment:
> On Apr 11, 2016, at 2:59 PM, Jon Wayne Parrott <notifications@github.com> wrote:
> 
> In tests/extras_appengine_auth_models_test.py <https://github.com/GoogleCloudPlatform/webapp2/pull/109#discussion_r59261220>:
> 
> > @@ -141,14 +141,14 @@ def test_user_token(self):
> >          auth_id = 'foo'
> >  
> >          token = m.create_auth_token(auth_id)
> > -        self.assertTrue(m.validate_auth_token(auth_id, token))
> > +        self.assertTrue(m.validate_auth_token(auth_id, token) != None)
> I'm seeing some of the assertNone but not the assertIsNotNones.
> 
> 

Yeah - well, that’s because I didn’t change them :/ And it’s assertIsNone.

Here’s hoping that everything is right this time and I can stop embarrassing myself with this trivial change :)